contents/tutorials/debug-missing-events.md

```diff
@@ -0,0 +1,41 @@
+---
+title: How to troubleshoot missing events
+date: 2023-08-08
+author: ["ian-vanagas"]
+showTitle: true
+sidebar: Docs
+featuredImage: ../images/tutorials/banners/tutorial-10.png
+tags: ['guides']
+---
+
+Missing events can be a pain to troubleshoot. This guide aims to help you diagnose and address the issue efficiently. There are three potential areas where the problem might originate:
+
+1. The library is being used incorrectly. (this is usually the issue)
+2. There is a bug within the library being used.
+3. Events are not ingested after being received by PostHog.
+
+## The library is being used incorrectly. (this is usually the issue)
+
+1. Visit [webhook.site](https://webhook.site/) and copy “Your unique URL”.
+2. Replace [posthog.com](https://app.posthog.com) (or [eu.posthog.com](https://eu.posthog.com) if using the EU Cloud) with “Your unique URL”.
+3. Make sure that the SDK sends a couple of events.
+4. See if the incoming events arrive as expected.
+
+## There is a bug within the library being used.
+
+1. Open the devtools and switch to the network tab.
+2. Look for the request that appears to be incorrect.
+3. [Send a support ticket via the app](https://app.posthog.com/home#supportModal) including the network request.
```

Comment:
The modal links take suffixes - make it select the `events ingestion` dropdown and `get support` or whatever. Also does it re-direct to eu.posthog.com for the modal usage, if they are a eu customer?

consider providing a helpful bug reporting format for users <- so I can tell them to send me the bug report in this format:
```
# Library:
# Library initialization:
# What was called:
# What was expected to happen:
# What actually happened:
```

For example:
```
# Library: 
posthog-python https://posthog.com/docs/libraries/python
# Library initialization:
from posthog import Posthog
posthog = Posthog(phc_yourAPIKey, host='app_or_eu.posthog.com')
# What was called:
posthog.capture(
    "Bob", 
    "user signed up", 
    {
        "login_type": "email", 
        "is_free_trial": "true"
    }
)
# What was expected to happen:
To see a network call like <x>
# What actually happened:
The network call looked like this, specifically "login_type" property was missing.
```